esi_leap/api/controllers/v1/contract.py

```diff
@@ -60,12 +61,17 @@ def get_one(self, contract_uuid):
     @wsme_pecan.wsexpose(ContractCollection, wtypes.text,
                          datetime.datetime, datetime.datetime, wtypes.text,
                          wtypes.text)
-    def get_all(self, project_id=None, start_date=None, end_date=None,
+    def get_all(self, project_id=None, start_time=None, end_time=None,
                 status=None, offer_uuid=None):
         request = pecan.request.context
         cdict = request.to_policy_values()
         policy.authorize('esi_leap:contract:get', cdict, cdict)
 
+        if start_time and end_time is None or end_time and start_time is None:
```

Comment:
This conditional is a little hard to read - can you put in parentheses here (and below)?

```
    if (start_time and end_time is None) or (end_time and start_time is None):
```